AppCenter/AppCenter.xcodeproj/project.pbxproj

```diff
@@ -737,6 +737,9 @@
 		F803BBF91E8E3989004B1E7A /* MSCustomPropertiesLog.m in Sources */ = {isa = PBXBuildFile; fileRef = F803BBF81E8E3989004B1E7A /* MSCustomPropertiesLog.m */; };
 		F803BC381E8E6927004B1E7A /* MSCustomPropertiesTests.m in Sources */ = {isa = PBXBuildFile; fileRef = F803BC371E8E6927004B1E7A /* MSCustomPropertiesTests.m */; };
 		F803BC3A1E8E6963004B1E7A /* MSCustomPropertiesLogTests.m in Sources */ = {isa = PBXBuildFile; fileRef = F803BC391E8E6963004B1E7A /* MSCustomPropertiesLogTests.m */; };
+		F82E4C6D217F159A00EDAB34 /* sqlite3.c in Sources */ = {isa = PBXBuildFile; fileRef = F82E4C6C217F159A00EDAB34 /* sqlite3.c */; settings = {COMPILER_FLAGS = "-w"; }; };
```

Comment:
Is it really required? I could run all testing without this one. Can you double check?

Reply:
`sqlite3` is required since we really use it.
It can be used by link system one (was used before) or any another implementation.
The latest official implementation has been added instead of system one (see [xcconfig](https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1184/files#diff-8a9c66c4d1bf5fc63394d7af037ec1bcR4)).

And yes, the tests passed with system implementation, but key thing here is official implementation can handle non-standard sql syntax (topic of the PR). So, this change allows detect the issue with previous implementation and makes the test meaningful (fail before, pass after).